instrumentation-core/spring/spring-core/spring-core.gradle

```diff
@@ -0,0 +1,27 @@
+ext {
+  minJavaVersionForTests = JavaVersion.VERSION_1_8
+}
+
+group = 'io.opentelemetry.spring.instrumentation'
+version = '0.0.1-SNAPSHOT'
+
+apply plugin: 'java'
+apply from: "$rootDir/gradle/java.gradle"
+
+sourceCompatibility = '8'
```

Comment:
It looks like we're targeting an old version of spring which is good since it means we can support more apps. But it means we should stick with default Java 7